tests/spring-boot/src/main/java/io/hawt/tests/spring/boot/SampleCamelRouter.java

```diff
@@ -0,0 +1,25 @@
+package io.hawt.tests.spring.boot;
+
+import org.apache.camel.builder.RouteBuilder;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class SampleCamelRouter extends RouteBuilder {
+
+    @Override
+    public void configure() throws Exception {
+        // Uncomment to enable the Camel plugin Debug tab
+        // getContext().setDebugging(true);
```

Comment:
@tadayosi tbh, I didn't notice any change when I uncommented this line - when it was commented out I was able to see Debug tab and do debugging. When I uncommented nothing changed for me. Or this supposed to be something else?

The screenshot made when the line was commented out
<img width="1720" alt="Screenshot 2023-02-07 at 15 04 27" src="https://user-images.githubusercontent.com/49488040/217240434-8cdccf77-d6f5-4609-80b6-0e4364dbe1f9.png">

Reply:
OK, since Camel 3.16 when you have `camel-debug` in deps it seems it's automatically enabled. So you can simply delete these 2 lines. I'll update the examples later.
https://camel.apache.org/manual/debugger.html#_camel_route_debugger_through_jmx

Reply:
Ok, I will remove those 2 lines, thanks